build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.5.1'
```

Comment:
Why update this in this PR?

Reply:
hmm, I don't know where this came from. I guess I pushed some update button in android studio, then it changed project build file on its own. I'll take it away.